libs/datasets/new_cases_and_deaths.py

```diff
@@ -72,36 +72,39 @@ def add_new_deaths(dataset_in: MultiRegionDataset) -> MultiRegionDataset:
 def spread_first_reported_value_after_stall(
     series: pd.Series, max_days_to_spread: int = 7
 ) -> pd.Series:
-    """Spreads first reported value after reported zeros.
+    """Spreads first reported value after reported zeros by dividing it evenly
+    over the prior days.
 
     Args:
         series: Series of new cases with date index.
-        max_days_to_spread: Maximum number of days to spread a single report.
+        max_days_to_spread: Maximum number of days to spread a single report. If
+            a stall exceeds this number of days, the reported value will be spread
+            over max_days_to_spread and the remaining zeros will be kept as
+            zeros.
     """
-    # Find points in the series that are either zeros or the first report after a string of zeros.
```

Comment:
This comment didn't help me understand the code any.. (where are we "finding" and what does that even mean)